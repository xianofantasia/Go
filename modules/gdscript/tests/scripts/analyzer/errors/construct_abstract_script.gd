const A = preload("./construct_abstract_script.notest.gd")

func test():
	var _a := A.new()
