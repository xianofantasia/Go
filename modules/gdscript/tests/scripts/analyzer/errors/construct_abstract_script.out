GDTEST_ANALYZER_ERROR
>> ERROR at line 4: Cannot construct abstract class "TestConstructAbstractScriptNotest".
